### PR TITLE
Development

### PR DESCRIPTION
**Pull request by osinakayah**

### CODE REVIEW

File: src/function.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+function getCeaserChar(arrayOfAlphabet, char, step) {
+  for(let i = 0; i < arrayOfAlphabet.length; i++){
+    const singleChar = arrayOfAlphabet[i]
+    if (singleChar === char) {
+<<<<<<< HEAD
+        return arrayOfAlphabet[(i + step)]
+=======
+      return arrayOfAlphabet[(i + step)]
+>>>>>>> d6cd0daa41bf45f9c1e7293084cb64c78f7848a4
```
Comment (heyapricot) [inline src/function.js]: Unresolved merge conflict?

**Comment by osinakayah** [comment]
I don't understand the feedback

**Comment by heyapricot** [comment]
Also: 
- Add a descriptive README.md
- Remove the `.idea` folder from the project
- Add a `.idea` rule to your `.gitignore` file

**Comment by dipto0321** [comment]
Hi @osinakayah and @sanelca ,

This project looks good. All test passed. This is from me 👍 . You can merge it.

Thanks 🙏

**Pull request merged**